### PR TITLE
Create basic react ui example using sdk

### PR DESCRIPTION
**Pull request by rpedroni**

### CODE REVIEW

File: examples/ui-search/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  StatHelpText,
+  Flex,
+  Box,
+  Heading,
+  Spacer,
+  ButtonGroup,
+  Divider,
+} from '@chakra-ui/react';
+
+const config = new Configuration({ network: 'local' });
```
Comment (Naguumo) [inline examples/ui-search/src/App.tsx]: If this is an example for users, shouldn't we be using 'goerli'?

File: examples/ui-search/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    setSearchResultOrgs(orgs);
+    setLoading(false);
+    setSearch('');
+  };
+
+  return (
+    <Container padding={10}>
+      <Flex alignItems="center" gap="2">
+        <Box p="2">
+          <Heading size="md">Art Blocks</Heading>
```
Comment (Naguumo) [inline examples/ui-search/src/App.tsx]: Why does this say Art Blocks?

**Review by Naguumo — CHANGES_REQUESTED** [review]
Minor changes requested!

File: examples/ui-search/src/App.tsx (hunk 2 above)
Comment (rpedroni) [reply examples/ui-search/src/App.tsx]: Both good points - prepping this up for Friday's demo, which is for Art Blocks. That being said, we should not merge this as is - I'll change this to be the "real" example values

**Review by Naguumo — APPROVED** [review]

**Pull request merged**